lib/rubocop/cop/metrics/utils/code_length_calculator.rb

```diff
@@ -153,6 +153,20 @@ def irrelevant_line?(source_line)
           def count_comments?
             @count_comments
           end
+
+          def omit_length(descendant)
+            parent = descendant.parent
+            return 0 if another_args?(parent)
+
+            [
+              parent.loc.begin.end_pos != descendant.loc.expression.begin_pos,
+              parent.loc.end.begin_pos != descendant.loc.expression.end_pos
+            ].count(true)
+          end
+
+          def another_args?(node)
+            node.send_type? && node.children.count > 3
```

Comment:
```suggestion
            node.send_type? && node.arguments.count > 1
```

is easier to understand

Reply:
Also better use `call_type?` as it also covers csend (&.) nodes

Reply:
Thank you, I've updated `another_args?` and add a test case.